project/Doc.scala

```diff
@@ -132,10 +133,22 @@ object UnidocRoot extends AutoPlugin {
       .getOrElse(sbtunidoc.ScalaUnidocPlugin)
 
   val akkaSettings = UnidocRoot.CliOptions.genjavadocEnabled
-    .ifTrue(Seq(javacOptions in (JavaUnidoc, unidoc) := {
-      if (JdkOptions.isJdk8) Seq("-Xdoclint:none")
-      else Seq("-Xdoclint:none", "--ignore-source-errors", "--no-module-directories")
-    }))
+    .ifTrue(Seq(
+      javacOptions in (JavaUnidoc, unidoc) := {
+        if (JdkOptions.isJdk8) Seq("-Xdoclint:none")
+        else Seq("-Xdoclint:none", "--ignore-source-errors", "--no-module-directories")
+      },
+      deployRsyncArtifacts := {
```

Comment:
`deployRsyncArtifacts +=` is used in Paradox, but here `:=`. Any risk that this overrides the other?

Reply:
Not currently, because these settings are applied to the root project 'manually', while the paradox settings are applied to the docs subproject via the local plugin. There is of course a risk that we would refactor this at some point and break this, but preventing that seems more complicated than it's worth right now.

Reply:
ok, I thought the order would be more independent if this also used `:+`, but it might be more to it

Reply:
Ah, we can use ++= if we load the DeployRsync earlier, doing that :+1: